components/dashboard/src/Login.tsx

```diff
@@ -170,72 +175,79 @@ export const Login: FC<LoginProps> = ({ onLoggedIn }) => {
                                         >
-                                            {iconForAuthProvider(ap.authProviderType)}
+                                            {iconForAuthProvider(providerFromContext.authProviderType)}
                                             <span className="pt-2 pb-2 mr-3 text-sm my-auto font-medium truncate overflow-ellipsis">
-                                                Continue with {simplifyProviderName(ap.host)}
+                                                Continue with {simplifyProviderName(providerFromContext.host)}
                                             </span>
                                         </button>
-                                    ))
-                                )}
-                                <SSOLoginForm
-                                    onSuccess={authorizeSuccessful}
-                                    singleOrgMode={!!authProviders.data && authProviders.data.length === 0}
-                                />
+                                    ) : (
+                                        authProviders.data?.map((ap) => (
+                                            <button
+                                                key={"button" + ap.host}
+                                                className="btn-login flex-none w-56 h-10 p-0 inline-flex"
+                                                onClick={() => openLogin(ap.host)}
+                                            >
+                                                {iconForAuthProvider(ap.authProviderType)}
+                                                <span className="pt-2 pb-2 mr-3 text-sm my-auto font-medium truncate overflow-ellipsis">
+                                                    Continue with {simplifyProviderName(ap.host)}
+                                                </span>
+                                            </button>
+                                        ))
+                                    )}
+                                    <SSOLoginForm
+                                        onSuccess={authorizeSuccessful}
+                                        singleOrgMode={!!authProviders.data && authProviders.data.length === 0}
+                                    />
+                                </div>
+                                {errorMessage && <ErrorMessage imgSrc={exclamation} message={errorMessage} />}
                             </div>
-                            {errorMessage && <ErrorMessage imgSrc={exclamation} message={errorMessage} />}
                         </div>
-                    </div>
-                    <div className="flex-none mx-auto h-20 text-center">
+                    )}
+                    <div className="flex-none mx-auto text-center px-4 pb-4">
                         <span className="text-gray-400">
                             By signing in, you agree to our{" "}
```

Comment:
**question**: Could we remove this if the setup is pending, as it doesn't make much sense to keep around? WDYT?